eng/pipelines/ci.yml

```diff
@@ -0,0 +1,91 @@
+# This pipeline is triggered only for pr
+trigger: none
+
+pr:
+  branches:
+    include:
+    - autorestv3
+
+variables:
+  NodeVersion: '10.x'
+  PythonVersion: '3.6'
+  TestFolder: '$(Build.SourcesDirectory)/test/azure'
```

Comment:
You need to do the "vanilla" folder too

Reply:
Not a problem. Working through understanding why the `.coverage` stuff isn't working.

Reply:
It seems coverage v5 is creating a lot of unstability...

Reply:
Yeah. I'm pinning coverage to 4.5.4 in the `ci` build. coverage5.0 is causing a bunch of issues.